akka-persistence/src/main/resources/reference.conf

```diff
@@ -243,6 +243,34 @@ akka.persistence.journal.leveldb-shared {
     }
 }
 
+akka.persistence.journal.proxy {
+  # Class name of the plugin.
+  class = "akka.persistence.journal.JournalProxy"
+  # Dispatcher for the plugin actor.
+  plugin-dispatcher = "akka.actor.default-dispatcher"
+  # Set this to on in the configuration of the ActorSystem
+  # that will host the target journal
+  start-target-journal = off
+  # The journal plugin config path to use for the target journal
+  target-journal-plugin = ""
```

Comment:
maybe add this comment in the conf file ("e.g. akka.persistence.journal.leveldb"), so it's clear we want the path, not the class of the plugin